api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -224,4 +224,34 @@
 								 baseTableName="encounter_diagnosis" baseColumnNames="changed_by"
 								 referencedTableName="users" referencedColumnNames="user_id" />
 	</changeSet>
+	<changeSet id="20182602-TRUNK-5349-0001" author="segunolalive">
+		<preConditions onFail="MARK_RAN">
+			<not>
```

Comment:
Instead of creating these as new changesets, just add them to the already merged changesets which were originally created. This is because those merged changes have not yet been released in any version of the OpenMRS platform.